src/orthogonalize.jl

```diff
@@ -12,9 +13,15 @@ function right_orthogonalize(
   # Start by getting the right eivenvector/eigenvalue of T
   # TODO: make a function `right_environments(::InfiniteMPS)` that computes
   # all of the right environments using `eigsolve` and shifting unit cells
-  λ⃗₁ᴿᴺ, v⃗₁ᴿᴺ, eigsolve_info = eigsolve(T, v₁ᴿᴺ, 1, :LM; tol=tol)
+
+  # original eigsolve function, switch to schur which enforces real
+  #λ⃗₁ᴿᴺ, v⃗₁ᴿᴺ, eigsolve_info = eigsolve(T, v₁ᴿᴺ, 1, :LM; tol, eager=true)
+  TT, v⃗₁ᴿᴺ, λ⃗₁ᴿᴺ, eigsolve_info = schursolve(T, v₁ᴿᴺ, 1, :LM, Arnoldi(; tol))
   λ₁ᴿᴺ, v₁ᴿᴺ = λ⃗₁ᴿᴺ[1], v⃗₁ᴿᴺ[1]
 
+  if size(TT, 2) > 1 && TT[2, 1] != 0
+    @warn("Largest transfer matrix eigenvector is not real?")
```

Comment:
Seems a bit funny to make this a question. Can't you check if it is real?

Reply:
If it isn't a definitive check, it would be better to phrase it as `"Largest transfer matrix eigenvector might not be real."`.

Reply:
I guess it should be "Non-unique largest eigenvector" (or the eigenvector shouldn't be real)

Reply:
I think what it really means is that it is not unique, this results in complex conjugate eigenvalue pairs (hence checking the `TT[2, 1]`, `TT[1:2,1:2] = [real(lambda) imag(lambda); -imag(lambda) real(lambda)]` or something like that).